screens/LoginScreen.js

```diff
@@ -120,7 +122,7 @@ class LoginScreen extends React.Component {
     if (props.user && props.user.token) {
       this.state = {
         ...this.state,
-        loading: true,
+        // loading: true,
```

Comment:
@Shizo0o0o Can you uncomment this line since it was just to workaround the occasional login problems?